Composer/packages/client/src/telemetry/AppInsightsClient.ts

```diff
@@ -6,14 +6,24 @@ import {
   TelemetryEvent,
   TelemetryEventTypes,
   TelemetrySettings,
-  persistedEvents,
   TelemetryEventName,
+  persistedEvents,
+  alwaysTrackEvents,
+  piiProperties,
 } from '@bfc/shared';
 import chunk from 'lodash/chunk';
+import fromPairs from 'lodash/fromPairs';
 
 import httpClient from '../utils/httpUtil';
 
 const BATCH_SIZE = 20;
+
+const blankPIIObject = fromPairs(piiProperties.map((p) => [p, null]));
```

Comment:
You don't need to remove PII in the client. We handle that on the server since that's were most of the PII is set.

Reply:
Fixed this in the latest version.